packages/app-project/src/components/ProjectHeader/hooks/useStores.js

```diff
@@ -18,6 +18,13 @@ function storeMapper(store) {
     }
   } = store
 
+  const availableLocales = languages?.map(current => {
+    let newLocale = current
+    if (current === 'zh-cn') newLocale = 'zh-CN'
+    if (current === 'zh-tw') newLocale = 'zh-TW'
+    return newLocale
```

Comment:
This function will need to be updated for each new language that has dialects eg. Brazilian Portugese. I'd consider generalising it to work with any two part language code.

```suggestion
    const [language, dialect] = current.split('-')
    if (dialect) {
      return `${language}-${dialect.toUpperCase()}`
    }
    return language
```